src/pages/Main.jsx

```diff
@@ -0,0 +1,158 @@
+import { useState, useEffect } from 'react'
+import styled from 'styled-components'
+import axios from 'axios'
+// import YouTube from 'react-youtube'
+import { Link } from 'react-router-dom'
+
+const Main = () => {
+	const [playlist, setPlaylist] = useState([])
+	useEffect(() => {
+		axios
+			.get(
+				'https://www.googleapis.com/youtube/v3/videos?part=snippet&chart=mostPopular&maxResults=4&key=AIzaSyCZmSMmjRPzSl8o8u6KUIR4Nf8J2v1DJGs'
+			)
+			.then((res) => {
+				console.log(res)
+				setPlaylist(res.data.items)
+			})
+			.catch(() => {})
+	}, [])
+
+	return (
+		<>
+			<VideoCard>
+				<div className="filterWrapper">
+					<div className="filterItem">전체</div>
+					<div className="filterItem">실시간</div>
+					<div className="filterItem">음악</div>
+					<div className="filterItem">뉴스</div>
+					<div className="filterItem">게임</div>
+					<div className="filterItem">수학</div>
+				</div>
+			</VideoCard>
+			<VideoList className="video_content">
+				<div className="videoComponent">
+					{playlist &&
+						playlist.map((i, idx) => {
+							return (
+								<div className="playlist" key={idx}>
+									<Link to={'/video/' + i.id}>
+										<img src={i.snippet.thumbnails.high['url']} alt="" />
+									</Link>
+									<div>
+										<Link to={'/video/' + i.id}>
+											<h1>{i.snippet.localized['title']}</h1>
+										</Link>
+									</div>
+								</div>
+							)
+						})}
+				</div>
```

Comment:
<div className="videoComponent">
					{isLoading ? (
						<p>isLoading...</p>
					) : (
						<>
							{data &&
								isSuccess &&
								data.map((i, idx) => {
									return (
										<div className="playlist" key={idx}>
											<Link to={'/video/' + i.id}>
												<img src={i.snippet.thumbnails.high['url']} alt="" />
											</Link>
											<div>
												<Link to={'/video/' + i.id}>
													<h1>{i.snippet.localized['title']}</h1>
												</Link>
											</div>
										</div>
									)
								})}
						</>
					)}
				</div>